acceptance/run_combination.go

```diff
@@ -0,0 +1,121 @@
+package acceptance
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/buildpacks/pack/internal/builder"
+	"github.com/buildpacks/pack/internal/style"
+	"github.com/pkg/errors"
+	"os"
+	"path/filepath"
+)
+
+const envAcceptanceSuiteConfig = "ACCEPTANCE_SUITE_CONFIG"
+
+type runCombo struct {
+	Pack              string `json:"pack"`
+	PackCreateBuilder string `json:"pack_create_builder"`
+	Lifecycle         string `json:"lifecycle"`
+}
+
+type resolvedRunCombo struct {
+	packCreateBuilderFixturesDir string
+	packFixturesDir              string
+	packPath                     string
+	packCreateBuilderPath        string
+	lifecyclePath                string
+	lifecycleDescriptor          builder.LifecycleDescriptor
+}
+
+func (c *runCombo) String() string {
+	return fmt.Sprintf("p_%s cb_%s lc_%s", c.Pack, c.PackCreateBuilder, c.Lifecycle)
+}
+
+func getRunCombinations() ([]runCombo, error) {
+	combos := []runCombo{
+		{Pack: "current", PackCreateBuilder: "current", Lifecycle: "default"}, // TODO: the current behavior for
```

Comment:
We should discuss and align on this. I think the code as-is preserves the current behavior.